Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/find.py

```diff
@@ -51,40 +51,12 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def main(cls, args, repository, reference='HEAD', **kwargs):
```

Comment:
This is just a location move, right?

Reply:
Yes! We're breaking out all of the `print` details and `git` makes this diff a bit weird.